it/auth/IndividualsStateBenefitsAuthSupportingAgentsAllowedISpec.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package auth
+
+import api.models.domain.TaxYear
+import api.services.DownstreamStub
+import play.api.http.Status.OK
+import play.api.libs.json.{JsValue, Json}
+import play.api.libs.ws.{WSRequest, WSResponse}
+
+class IndividualsStateBenefitsAuthSupportingAgentsAllowedISpec extends AuthSupportingAgentsAllowedISpec {
+
+  val callingApiVersion = "1.0"
+
+  val supportingAgentsAllowedEndpoint = "create-benefit"
+
+  private val taxYear = TaxYear.fromMtd("2019-20")
+
+  val mtdUrl = s"/$nino/${taxYear.asMtd}"
+
+  val requestBodyJson: JsValue = Json.parse(
+    s"""
+       |{
+       |  "benefitType": "incapacityBenefit",
+       |  "startDate": "2019-01-01",
+       |  "endDate": "2020-06-01"
+       |}
+      """.stripMargin
+  )
+  val benefitId: String = "b1e8057e-fbbc-47a8-a8b4-78d9f015c253"
+
+  val responseJson: JsValue = Json.parse(
```

Comment:
This could just go into the "val maybeDownstreamResponseJson", but inlined as "Some(Json.parse(...))" - doesn't need to be a separate "val responseJson".